content/_index.html

```diff
@@ -294,27 +294,28 @@ <h1>Irresistible Influence</h1>
 		<p><a href="https://youtu.be/Psu_iF_hDQ0">Watch live starting at <time datetime="2023-08-04T18:30-07:00">6:30pm Pacific</time>!</a></p>
 	</div> -->
 
+	<p>Thank you to everyone who attended Irresistible Influence 2023!</p>
 	<p>Recordings are available on our <a href="https://www.youtube.com/playlist?list=PLqVg8iJjuJ4cQ4ih5p2siEtzvEIDWK4CO">YouTube channel's 2023 playlist</a>. Please subscribe to the <a href="https://www.youtube.com/channel/UCAinceLJDt98CJWP4hbzn2A">channel</a> and click the bell icon to be notified.</p>
 	<p data-before="2023-08-02 GMT-0700">Check the <a href="/archives">archives</a> for recordings from previous meetings!</p>
 </section>
-<section class="quote_block" id="themescripture">
+<!-- <section class="quote_block" id="themescripture">
 	<p>And I, if I am lifted up from the earth, will draw all peoples to Myself.</p>
 	<p class="source_ref">John 12:32</p>
 	<p>As we are drawn by Christ's Irresistible Influence, He changes us to reflect His influence to those around us. "A true, unselfish, godly life carries an almost irresistible influence."</p>
 	<p class="source_ref"><a href="https://egwwritings.org/read?panels=p15.1501&index=0#highlight=15.1501|0">Christ's Object Lessons, p340</a></p>
-</section>
+</section> -->
 <section>
 	<p>Each year GYC West hosts a young adult retreat to gain strength in the faith, network safely with like-minded believers, and seek a closer communion with our Father.
-		For 2023 the theme is "Irresistible Influence."
-		More details about this event and the theme will come soon.</p>
-	<p>Join us in 2023 to come away from your work and routines and seek revival together.</p>
+		For 2024 the theme will be "Turning Hearts."
+		More details about this event and the theme to come.</p>
+	<p>Join us in 2024 to come away from your work and routines and seek revival together.</p>
 </section>
-<section id="speakers">
+<!--<section id="speakers">
```

Comment:
SGML comments require a space between the dashes and any content.

Reply:
Oops, missed that.

Reply:
Ok, I fixed it.